Makefile

```diff
@@ -47,6 +49,11 @@ build: vendor build-css
 	# Tie in content from external repositories
 	./build-external-repositories.sh
 
+	# Generate the Control Plane K8s API reference documentation.
+	docker run \
+		-v ${PWD}/build/content/reference/cp-k8s-api:/opt/crd-docs-generator/output \
+		quay.io/giantswarm/crd-docs-generator:1c23ac7c5dabec1aa70288c91d06a5c9d3863095
+
```

Comment:
The idea is to use `:latest` here instead of a specific commit, once https://github.com/giantswarm/crd-docs-generator/pull/1 has been merged into master.